diff --git a/package/base-files/files/bin/config_generate b/package/base-files/files/bin/config_generate
index 8e5e120..bbeb20a 100755
--- a/package/base-files/files/bin/config_generate
+++ b/package/base-files/files/bin/config_generate
@@ -163,8 +163,8 @@ generate_network() {
 		static)
 			local ipad netm prefix
 			case "$1" in
-				lan) ipad=${ipaddr:-"192.168.1.1"} ;;
-				*) ipad=${ipaddr:-"192.168.$((addr_offset++)).1"} ;;
+				lan) ipad=${ipaddr:-"10.0.0.1"} ;;
+				*) ipad=${ipaddr:-"10.0.$((addr_offset++)).1"} ;;
 			esac
 
 			netm=${netmask:-"255.255.255.0"}
diff --git a/package/base-files/files/etc/rc.local b/package/base-files/files/etc/rc.local
index 5639477..560109c 100644
--- a/package/base-files/files/etc/rc.local
+++ b/package/base-files/files/etc/rc.local
@@ -1,4 +1,9 @@
 # Put your custom commands here that should be executed once
 # the system init finished. By default this file does nothing.
 
+/usr/bin/AdGuardHome  --work-dir /etc/config/adg --config /etc/config/adg/AdGuardHome.yaml >/dev/null 2>&1 &
+/usr/bin/mosdns service start >/dev/null 2>&1
+
+for led in /sys/class/leds/*; do echo 0 > $led/brightness; done
+
 exit 0
diff --git a/target/linux/mediatek/dts/mt7981b-cudy-tr3000-v1.dts b/target/linux/mediatek/dts/mt7981b-cudy-tr3000-v1.dts
index d65e235..0deb856 100644
--- a/target/linux/mediatek/dts/mt7981b-cudy-tr3000-v1.dts
+++ b/target/linux/mediatek/dts/mt7981b-cudy-tr3000-v1.dts
@@ -22,5 +22,5 @@
 };
 
 &ubi {
-	reg = <0x5c0000 0x4000000>;
+	reg = <0x5c0000 0x7a40000>;
 };
